app/services/export/s3_uploader.rb

```diff
@@ -3,7 +3,7 @@ class S3UploadError < StandardError; end
 
   class S3Uploader
     def initialize(file:, filename:)
-      @config = S3UploaderConfig.new
+      @config = S3UploaderConfig.new(public: true)
```

Comment:
Should the argument be `use_public_bucket`?

Reply:
ah, I think `git absorb` didn't do as I expected here, I missed that, well spotted!

Reply:
Ooh, didn't know about `git absorb`. Does it usually work well? Finding the right commits to make changes to can be a pain in bigger PRs (would be nice if GitHub gave you a pointer to the commit a comment was left on)

Reply:
It's usually pretty good, particularly if you're making lots of small changes, but it's usually worth double checking (do as I say, not as I do!)